src/test/java/se/gu/spraakbanken/fcs/endpoint/korp/KorpEndpointSearchEngineTest.java

```diff
@@ -337,7 +337,8 @@ public void search1() throws SRUException, SRUConfigException, XMLStreamExceptio
 	    xmlStreamWriter.flush();
 	    xmlStreamWriter.close();
 	} catch (Exception ex) {
-	    xmlStreamWriter.close();
+		// TODO: throwing an error that blocks compilation, unsure how to proceed
+	    //xmlStreamWriter.close();
```

Comment:
Okey for now. Probably needs another try-catch wrapper. Also happened for me with Ubuntu + OpenJDK 11 / Maven 3.6.3.